client/package.json

```diff
@@ -89,7 +89,8 @@
     "webpack-cli": "^3.3.1",
     "webpack-dev-server": "^3.3.1",
     "webpack-merge": "^4.2.1",
-    "xhr2": "^0.1.4"
+    "xhr2": "^0.1.4",
+    "xmlhttprequest": "1.8.0"
```

Comment:
We are already using `xhr2`, is there a reason to add another dependency?

Reply:
I only see one dependency to xhr2, or am I missing something?

Reply:
The question is do we need both `xhr2` and `xmlhttprequest` and can't we just use one of them?

Reply:
Now I see what you meant. I will take a look into it.

Reply:
It was indeed unnecessary. Changed to xhr2.